tests/data_context/test_data_context.py

```diff
@@ -1276,151 +1231,6 @@ def test_build_batch_kwargs(titanic_multibatch_data_context):
     assert {"Titanic_1912.csv", "Titanic_1911.csv"} == set(paths)
 
 
-def test_existing_local_data_docs_urls_returns_url_on_project_with_no_datasources_and_a_site_configured(
```

Comment:
No changes to these tests - consolidated all DataContext docs related api tests into existing docs focused testing module.